### PR TITLE
add extra statistics

### PR DESCRIPTION
**Pull request by andreas-ibm**
hullo,

needed some extra statistics, added them :-)
changed some ints to uints since the ping response is an unsigned int

### CODE REVIEW

File: src/ESP8266Ping.impl.h [hunk 1]
```diff
@@ -98,7 +114,7 @@ void PingClass::_ping_recv_cb(void *opt, void *resp) {
     if (_success + _errors == _expected_count) {
         _avg_time = _success > 0 ? _avg_time / _success : 0;
 
-        DEBUG_PING("Avg resp time %d ms\n", _avg_time);
+        DEBUG_PING("Resp times min %d, ave %d, max %d ms\n", _min_time, _avg_time, _max_time);
```
Comment (dancol90) [inline src/ESP8266Ping.impl.h]: Small typo, here it should be `avg`!

File: src/ESP8266Ping.h [hunk 2]
```diff
@@ -40,7 +40,10 @@ class PingClass {
     bool ping(IPAddress dest,   unsigned int count = 5);
     bool ping(const char* host, unsigned int count = 5);
 
+    int minTime();
     int averageTime();
+    int maxTime();
+
```
Comment (dancol90) [inline src/ESP8266Ping.h]: I would remove the extra empty tile, just to keep the same codestyle

**Review by dancol90 — CHANGES_REQUESTED** [review]
Thanks for the submission! Nice work, I will accept the PR as soon as you fix these two small typos.

**Comment by andreas-ibm** [comment]
Think I've resolved them?

**Review by dancol90 — APPROVED** [review]

**Comment by dancol90** [comment]
Yes you did!

**Pull request merged**